Exit code: 1
---
Stdout:
======================= Schemathesis test session starts =======================
Schema location: http://127.0.0.1/schema.yaml
Base URL: http://127.0.0.1/api
Specification version: Open API 3.0.2
Random seed: 42
Workers: 1
Collected API operations: 2
Collected API links: 0
Report to Schemathesis.io: ENABLED
API probing: SUCCESS
Schema analysis: SUCCESS

GET /api/failure F                                                        [ 50%]
GET /api/success .                                                        [100%]

=================================== FAILURES ===================================
_______________________________ GET /api/failure _______________________________
1. Test Case ID: <PLACEHOLDER>

- Server error

[500] Internal Server Error:

    `500: Internal Server Error`

Reproduce with: 

    curl -X GET http://127.0.0.1/api/failure

=============================== SCHEMA ANALYSIS ================================

Success

Analysis took: 1.42ms

No extensions have been applied.

=================================== SUMMARY ====================================

Performed checks:
    not_a_server_error                    1 / 3 passed          FAILED 

Note: Use the 'X-Schemathesis-TestCaseId' header to correlate test case ids from failure messages with server logs for debugging.

Note: To replicate these test failures, rerun with `--hypothesis-seed=42`

Compressed report size: XX KB
Uploading reports to http://127.0.0.1 ...
Upload: ERROR

We apologize for the inconvenience. This appears to be an internal issue.
Please, consider reporting the following details to our issue tracker:

  https://github.com/schemathesis/schemathesis/issues/new?labels=Status%3A%20Needs%20Triage%2C+Type%3A+Bug&template=bug_report.md&title=%5BBUG%5D

Response: '"Content-Type error"'
Status: 400
Headers: {'X-Foo': 'Bar'}

Tip: Please update your CLI to the latest version and try again.

========================= 1 passed, 1 failed in 1.00s ==========================
